packages/@okta/vuepress-site/docs/reference/api/oie-policies/index.md

```diff
@@ -0,0 +1,778 @@
+
+API endpoints for listing, getting, creating, updating and deleting Policy and Rule objects function the same way in Okta Identity Engine as they do in the existing Okta `/policies` API; only the objects used are different, with Okta Identity Engine introducing a set of new Policy and Rule objects. See the [Okta Identity Engine Policy Objects](#okta-identity-enging-policy-objects) section of this document for descriptions of the objects.
+
+Each Policy object, as well as a Rule for it, needs to exist, in order for Okta Identity Engine to function. One of each is created by default when Okta Identity Engine is enabled for an org.
+
+A `/mappings` endpoint is used to set which Policies apply to which Apps. See the [Mappings Between Apps and Policies](#mappings-between-apps-and-policies) section of this document for a description of how to set mappings.
+
+## Policies API Operations
+
+API endpoints for creating, getting, and updating Policy and Rule objects function the same way in Okta Identity Engine as they do in the existing Okta `/policies` API. The API endpoints support the following operations:
+
+ * Get Policy
+ * List Policies
+ * Get Rule
+ * List Rules
+ * Create Policy
+ * Create Rule
+ * Update Policy
+ * Update Rule
+ * Delete Policy
+ * Delete Rule
+ 
+### Sample Operation: Get Policy
+
+<ApiOperation method="get" url="/policies/${policyId}" />
+
+Given a `policyId` identifier, returns a [Policy Object](#okta-identity-engine-policy-objects).
+
+#### Request Body
+
+| Property | Type   | Description                          |
+|----------|--------|--------------------------------------|
+| policyId | String | The unique identifier of the Policy. |
+
+#### Request Query Parameters
+
+None
+
+#### Response Body
+
+A [Policy Object](#okta-identity-engine-policy-objects).
+
+#### Usage Example
+
+##### Request
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/policies/${policyId}"
+```
+
+##### Response
+
+```json
+{
+    "id": "rs10xzYOyK9ux6v70g4",
+    "name": "Default Policy",
+    "type": "Okta:IdpRouting",
+    "status": "ACTIVE",
+    "default": true,
+    "_links": {
+        "self": {
+            "href": "https://idx.okta1.com/api/v1/policies/rst10xzYOyK9ux6v70g4",
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PUT"
+                ]
+            }
+        },
+        "rules": {
+            "href": "https://idx.okta1.com/api/v1/policies/rst10xzYOyK9ux6v70g4/rules",
+            "hints": {
+                "allow": [
+                    "GET"
+                ]
+            }
+        }
+    }
+}
+```
+
+### Syntax for All Other Operations
+
+See <https://developer.okta.com/docs/reference/api/policy/#policy-api-operations> for request syntax for all other operations.
+
+## Okta Identity Engine Policy Objects
```

Comment:
`Okta Identity Engine Policy Objects` -> `Okta Identity Engine Policy And Rule Objects`

Reply:
Made the change